contracts/interfaces/IHATVault.sol

```diff
@@ -73,6 +73,8 @@ interface IHATVault is IERC4626Upgradeable {
 
     /**
     * @notice Initialization parameters for the vault
+    * @param name The vault's name (concatinated as "HATs Vault " + name)
+    * @param symbol The vault's symbol (concatinated as "HAT" + symbol)
```

Comment:
```suggestion
    * @param symbol The vault's symbol (concatenated as "HAT" + name)
```

Reply:
Why? they are different the name and symbol and we pass both

Reply:
right, so just the typo